client/app/scripts/verifiedpixel-imagelist/module.js

```diff
@@ -520,7 +520,24 @@
                     filemeta.gpsinfo.gpsimgdirection[0] /
                     filemeta.gpsinfo.gpsimgdirection[1]
                 ).toFixed(3);
-                converted.markerdirection = Math.ceil(converted.gpsdirection / 10) * 10; 
+                converted.markerdirection = Math.ceil(converted.gpsdirection / 10) * 10;
+                if (filemeta.lensmodel) {
+                    var cameraface = filemeta.lensmodel;
+                    let camfacearr = [];
+                        camfacearr = cameraface.split(" ");
+                    for (let i of camfacearr) {
```

Comment:
maybe not use "let" just yet, as it is highly likely to cause bowser compatibility issues